### PR TITLE
Move crossing map and list links to header

### PR DESCRIPTION
**Pull request by briaguya**
After looking through the wires I realized instead of having sub-links on the crossing updates page, we should have links for the map and list in the header. I updated the header and routes accordingly.

### CODE REVIEW

File: frontend/src/Dashboard/CrossingMapPage/CrossingMapPage.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import CrossingMap from '../../Map/CrossingMap'; 
+import './CrossingMapPage.css';
+
+class CrossingMapPage extends Component {
+
+  render() {
+    const envelope = JSON.parse(this.props.currentUser.communityByCommunityId.viewportgeojson);
+    var viewport = [
```
Comment (mateoclarke) [inline frontend/src/Dashboard/CrossingMapPage/CrossingMapPage.js]: we should use `const` not `var`, right?

File: frontend/src/Dashboard/Header/Header.js [hunk 2]
```diff
@@ -24,8 +24,11 @@ class Header extends React.Component {
         <div className="Header__main">
           <h1 className="Header__h1">{"CTXfloods Dashboard"}</h1>
           <ul className="Header__tabs">
-            <li className={pathname.endsWith('crossings') ? 'Header__tab--active' : 'Header__tab'}>
-              <Link to="/dashboard/crossings">Crossing Updates</Link>
+            <li className={pathname.endsWith('crossings/list') ? 'Header__tab--active' : 'Header__tab'}>
+              <Link to="/dashboard/crossings/list">Crossing List</Link>
+            </li>
+            <li className={pathname.endsWith('crossings/map') ? 'Header__tab--active' : 'Header__tab'}>
```
Comment (mateoclarke) [inline frontend/src/Dashboard/Header/Header.js]: I'm probably wrong, but this logic feels a little brittle. If we change the pathnames, how would this fail?

Comment (briaguya) [reply frontend/src/Dashboard/Header/Header.js]: hmm, what would you suggest as a better solution?

File: frontend/src/Dashboard/Header/Header.js [hunk 3]
```diff
@@ -24,8 +24,11 @@ class Header extends React.Component {
...
+              <Link to="/dashboard/crossings/map">Crossing Map</Link>
             </li>
             <li className={pathname.endsWith('addcrossing') ? 'Header__tab' : 'Header__tab'}>
```
Comment (mateoclarke) [inline frontend/src/Dashboard/Header/Header.js]: Sorry I didn't see this at first. It looks like this logic will always return `Header__tab`. Never `--active`.

File: frontend/src/Dashboard/Header/Header.js (hunk 2 above)
Comment (mateoclarke) [reply frontend/src/Dashboard/Header/Header.js]: Yeah, I can't think of a better solution myself. Perhaps an option would be to pull out the pathnames and titles as key/values in an Array of objects that can be edited in one place. And then we `.map()` over them to create the `<link>` elements with its nested `<Link>`.

Meh, maybe over-engineering this non-issue?

**Pull request merged**